kickoff/jsonexport.py

```diff
@@ -228,6 +228,13 @@ def updateLocaleWithVersionsTable(product):
     buildsVersionLocales = generateLocalizedBuilds(buildsVersionLocales,
                                                    esr_releases[0][2],
                                                    esr_releases[0][0] + "esr")
+
+    esr_next = getFilteredReleases(product, ["esr"], lastRelease=True, withL10N=True, esrNext=True)
+    if esr_next:
+        buildsVersionLocales = generateLocalizedBuilds(buildsVersionLocales,
+                                                       esr_next[0][2],
+                                                       esr_next[0][0] + "esr")
```

Comment:
I think this will fail (IndexError) if you set `config.ESR_NEXT=""`. Maybe add a guard?